platform/openide.util.ui/nbproject/project.properties

```diff
@@ -16,8 +16,8 @@
 # under the License.
 
 javac.compilerargs=-Xlint -Xlint:-serial -Xlint:-processing
-javac.source=1.8
-javac.target=1.8
+javac.source=1.9
+javac.target=1.9
```

Comment:
https://github.com/apache/netbeans/issues/4904 is currently preventing to bump the module to >8

some tests which depend on this module fail on JDK 11.

but lets say the remaining tests would be fixed or disabled by someone:
 - change this to `javac.relese=11`
 - and remove the `OpenIDE-Module-Java-Dependencies` manifest entry gain since it is automatically added now

Reply:
https://github.com/apache/netbeans/pull/7502 is working on it

Reply:
now fixed

Reply:
Thank you; now the tests pass in this module indeed!

Reply:
property and manifest entry needs to be adjusted still